department-of-veterans-affairs/caseflow
# Preparing the case details page for AMA appeals

### Description
The case details page is currently tailored for legacy appeals and tasks. This makes a few changes to bring AMA appeals and tasks in line.

In particular it does two things:
1) Starts displaying document id for in the case snapshot for AMA appeals
1) Simplifies the task section to only show the task, assigned by, instructions, and in some cases the preparer of the decision.

While working on this we also made the document id click to copy since judges have asked for that.

### Testing Plan
1. Go to BVASCASPER's queue. The first appeal is an AMA appeal. Click on it.
1. Ensure things look right on this case detail page.
1. Go through a checkout flow and assign it to another judge
1. Go to that judge's queue and ensure their case details page looks correct and the document id is present.
